### PR TITLE
docs(general): Update README and CONTRIBUTING

### PR DESCRIPTION
**Pull request by bbender**
Update `README.md` to streamline the getting started information and move build and test instructions into `CONTRIBUTING.md`.

### CODE REVIEW

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -1,20 +1,96 @@
...
+- `CISCOSPARK_REDIRECT_URI`
+- `CISCOSPARK_SCOPE`
+
+You can get these values by registering a new integration on the [Spark for Developers](https://developer.ciscospark.com/add-integration.html) portal.
+
+### Building the SDK
+
+Fork the [spark-js-sdk](https://github.com/ciscospark/spark-js-sdk/) repository and `git clone` your fork:
+
+    git clone https://github.com/your-username/spark-js-sdk.git
```
Comment (ianwremmel) [inline CONTRIBUTING.md]: It looks like you're missing backticks for these commands.

File: CONTRIBUTING.md [hunk 2]
```diff
@@ -1,20 +1,96 @@
...
+
+Install tooling dependencies with:
+
+    npm install
+
+Specify which package to build by using the `PACKAGE` environment variable. To build the entire SDK, use `ciscospark`:
```
Comment (ianwremmel) [inline CONTRIBUTING.md]: The "entire" sentence is not accurate.

File: CONTRIBUTING.md [hunk 3]
```diff
@@ -1,20 +1,96 @@
...
+
+    PACKAGE=PACKAGENAME npm run grunt:package -- express:test test:node
+
+#### Run Browser Tests
+Keeps the browser open so that you can reload set break points and reload the page
+
+    KARMA_DEBUG=true PACKAGE=PACKAGENAME npm run grunt:package -- express:test test:browser
+
+### Git Commit Guidelines
+
```
Comment (ianwremmel) [inline CONTRIBUTING.md]: I just discovered https://conventionalcommits.org. We should try to rely on that when possible, but I haven't read through all of it yet.

File: CONTRIBUTING.md (hunk 1 above)
Comment (bbender) [inline CONTRIBUTING.md]: The `git clone` command will be shown as a code block, though it's not fenced by back-ticks.

File: CONTRIBUTING.md (hunk 2 above)
Comment (bbender) [inline CONTRIBUTING.md]: What would be a better command to build the SDK without building each package individually?

Comment (bbender) [inline CONTRIBUTING.md]: What would be a better command to build the SDK without building each package individually?

**Comment by bbender** [comment]
Merged 26-Apr-2017.

**Pull request closed without merging**